### PR TITLE
Remove fourier from ioSPI

### PR DESCRIPTION
**Pull request by geoffwoollard**
Together with https://github.com/compSPI/simSPI/pull/66 resolves https://github.com/compSPI/ioSPI/issues/45

### CODE REVIEW

File: tests/test_starfile.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 def normalized_mse(a, b):
     """Return normalized error between two numpy arrays."""
-    return np.sum((a - b) ** 2) ** 0.5 / np.sum(a ** 2) ** 0.5
+    return np.sum((a - b) ** 2) ** 0.5 / np.sum(a**2) ** 0.5
```
Comment (geoffwoollard) [inline tests/test_starfile.py:16]: this is from someone else. I had to use my local black (turn off the precommit hook) to have it change this. then it passed linting. someone (capstone student or harshit) is likely having the same issue as I am.

Comment (ninamiolane) [reply tests/test_starfile.py:16]: Yes it seems that there has been a black update

**Review by fredericpoitevin — APPROVED** [review]

**Comment by fredericpoitevin** [comment]
The codevoc/project fails just because there are less files to test (the numerator decreases but the denominator kept constant). I think we can merge.

**Review by ninamiolane — APPROVED** [review]
LGTM, thanks!

**Comment by ninamiolane** [comment]
@geoffwoollard conflicts must be resolved, otherwise good to merge, thank you!

**Pull request merged**